notebooks/hybrid_rag_pipeline_with_breakpoints.ipynb

```diff
@@ -0,0 +1,318 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #29.        rag_pipeline.add_component(instance=PromptBuilder(template=prompt_template, required_variables=['documents', 'question']), name="prompt_builder")
</pre><p>You could also use "*" for required_variables to shorten the line.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-cookbook/pull/186/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>